content/participate/index.html.haml

```diff
@@ -181,15 +181,21 @@ noanchors: true
     .bg-img.bg-img-design
     .content
       %h3 DESIGN
-      %p.p1 As it is intended for daily use by finicky web developers, design is essential
+      %p.p1 As it is intended for daily use by finicky web developers, design is essential.
       %a.btn.btn-design.bl{ :href => "/participate/design"} Read More
   .card
     .bg-img.bg-img-review
     .content
       %h3 REVIEW
       %p.p1 Help review changes to code or documentation.
       %a.btn.btn-review.bl{ :href => "/participate/review-changes"} Read More
- 
+  .card
+    .bg-img.bg-img-review
+    .content
+      %h3 Donate
```

Comment:
Optionally retain consistency with the other panels in the page.  They all start with a word that is all caps.
```suggestion
      %h3 DONATE
```